content/legal/_index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Legal notice
+menu:
+  main:
+    parent: 'Resources'
+    name: 'Legal notice'
+---
+
+# Legal notice
+
+The website [www.prestashop-project.org/](https://www.prestashop-project.org/) is owned by PrestaShop SA, corporation company with a capital of 380 645,55 euros.
+The headquarters are located at 4, rue Jules Lefebvre 75009 Paris, France.
+
+- CS registration number: B 497 916 635
+- VAT number: FR66497916635
+- Publication director: Paolo Fiorelli
+- Phone number: +33 (0)1 76 23 25 25
+- E-mail: contact@prestashop.com
+- Hosting: Jaguar Network – 71 Avenue André Roussin BP 50067 – 13321 Marseille Cedex 16
+- Number : +33 (0)1 83 65 99 00
+
+&copy; 2007-2022 PrestaShop SA. PrestaShop is a registered trademark.
+
+### Preamble
+
+These general terms and conditions of use (the "T&Cs") are intended to define the conditions of access and use of the website [https://www.prestashop-project.org/](https://www.prestashop-project.org/) (the "Website"), published by PrestaShop, a limited liability company with a share capital of €380,645.55, registered in the Paris Trade and Companies Register under number 497 916 635 and whose registered office is located at 4, rue Jules Lefebvre - 75009 Paris ("PrestaShop"), as well as the offer and acquisition of products and services of the latter, which are available on the Website.
+
+The user of the Website (the "User") is informed that by browsing the Website he or she has (i) had the opportunity to access the T&Cs and the documents to which they refer and (ii) has agreed to abide by them.
+
+If the User does not accept the T&Cs, he must not use the Website.
+The present T&Cs can be modified by PrestaShop at any time, without notice, to take into account the evolution of the Website and the applicable regulations. The new version of the T&Cs will be made available on the Website.
+
+The User is invited to consult them regularly.
+
+The use of the Website is free.
+
+**1. Purpose of the Website.**
+
+The purpose of the Website is to provide documentation to the PrestaShop project contributors.
+
+The PrestaShop project is a universal open-source software platform to build your e-commerce solution.
+ 
+**2. Intellectual Property.**
+
+All Website elements and their design, selection, presentation and organization are protected by intellectual property rights owned by PrestaShop. 
```

Comment:
To be updated, [our theme](https://github.com/prestashop/ps-org-theme/) is based on another one:

> © 2020 Designed by [Themefisher](https://themefisher.com/) & Developed by [Gethugothemes](https://gethugothemes.com/)
> 
> Code License: Released under the [MIT](https://github.com/themefisher/bigspring-hugo-startup-theme/blob/master/LICENSE) license.

Reply:
Resolving this also.